src/M6Web/Bundle/ElasticsearchBundle/Connection/CurlMultiConnectionDecorator.php

```diff
@@ -45,11 +45,14 @@ public function performRequest($method, $uri, $params = null, $body = null, $opt
             throw $e;
         }
 
+        $took = $this->extractTookFromResponse($response);
+
         $this->dispatchEvent(
             $method,
             $uri,
-            $response['requestInfo']['http_code'],
-            $response['requestInfo']['total_time'] * 1000  // Convert from seconds to milliseconds
+            $response['status'],
+            $response['info']['total_time'] * 1000, // Convert from seconds to milliseconds
```

Comment:
The same as @mguillermin just above : why this modification ?
Is there a Elasticsearch version update ?